src/clr-angular/forms/datepicker/daypicker.html

```diff
@@ -1,21 +1,46 @@
 <div class="calendar-header">
+    <div aria-live="polite" class="clr-sr-only">
+        The current month is {{ calendarMonth }}.
```

Comment:
This cannot be hard coded. We need to make this a translatable string. The problem is, this has a variable unlike any other string today.

Reply:
I've updated to use a single string with a CommonStrings service that can replace parts of the string.